cmd/gen.go

```diff
@@ -0,0 +1,36 @@
+// +build ignore
+
+// This program generates all-colors.go, which allows us to compile
+// our all-colors.json dataset inside of our Go binary, for easy access.
+//
+// This file is designed to be only be invoked with go:generate
+package main
```

Comment:
Do you think it would be valuable to test individual modules separately or is testing `main` enough?

Reply:
I tried a couple different things before landing on my current setup. I think it's fine for now, but maybe if the number of tests grows, I'd want to look into splitting it up somehow.